site/installation/migration.md

```diff
@@ -193,21 +195,28 @@ CRD. Please refer to the detailed guide on [migrating inventory objects] to the
 
 ## Migration Steps
 
-Based on the changes discussed above, this section walks you through an end to
-end workflow to migrate an [example kpt package] which is compatible with
-`v0.39` version of kpt, and make it compatible with kpt `v1.0`.
+### For Package Publishers
 
-### Automated portion of migration
+Based on the changes discussed above, this section walks package publishers through an end to
+end workflow to migrate their existing packages which are compatible with
+`v0.39` version of kpt, and make them compatible with kpt `v1.0`.
 
-Invoke `gcr.io/kpt-fn/fix` function from the root directory of the target kpt
-package.
+#### Automated portion of migration
+
+Since you are the package publisher, you are expected to have the latest version
+of published package on your local. If you do not already have it, you can [git clone]
+the latest version of remote package on to your local.
+
+Invoke `gcr.io/kpt-fn/fix` function from the root directory of the kpt package.
 
 ```shell
-# you must be using v1.0 version of kpt
+$ cd path/to/your/package
```

Comment:
Rule 2: https://github.com/GoogleContainerTools/kpt/blob/main/docs/style-guides/docs.md

Audit everywhere else.

Reply:
Updated. Used PKG_PATH. We can't have exact command here as the package path is local to the users.